baseline.yaml

```diff
@@ -192,6 +192,34 @@ criteria:
     scorecard_probe:
       - # TODO
 
+  - id: OSPS-AC-08
+    maturity_level: 1
+    category: Access Control
+    criteria: |
+      The project's version control system MUST require
+      second-party approval of commits before merging
+      changes into the primary branch or release branches.
+    objective: |
+      Ensure that changes to the project's codebase have
+      been reviewed and approved by two distinct
+      individuals before being merged.  This separation
+      of duties raises the difficulty of malicious commits
+      being merged into the project's codebase.
```

Comment:
Do you mean that two reviewers are required, or does the "two distinct individuals" include the person who wrote the change? I think you intended the latter. Not quite sure what I'd suggest to clarify yet, but I'll think about it more.

Reply:
Oh, if I'd looked at the implementation, I'd see that you say "at least one approval" :-)

Reply:
I'm assuming the person(s) who authored the change implicitly approve of it.

Interestingly, when working in pair authoring setups with `co-authored-by`, you can sometimes end up needing *more than two* approvals with the default GitHub approval mechanism.  That's really just a "you need other tooling than GitHub" situation (for which things like Prow exist).

I'm happy to accept suggestions on verbiage; I'm not as familiar with what the standard language may be for these types of controls.  (I care about the implementation of the control, but almost zero about the naming and standardization...)